src/sas/qtgui/MainWindow/MainWindow.py

```diff
@@ -61,8 +63,11 @@ def run_sasview():
     from sas.logger_config import SetupLogger
     SetupLogger(__name__).config_development()
 
-    # initialize OpenCL setting
-    from sas import get_custom_config
+    # initialize sasmodels settings
+    from sas import get_custom_config, get_user_dir
+    if "SAS_DLL_PATH" not in os.environ:
```

Comment:
Would it make sense to add `SAS_DLL_PATH` to the custom config file, like `SAS_OPENCL` does below?

Reply:
It's not really an option that the user needs to change, and there is no support for changing it in the gui, so no need to have it in the user config file.